_components/scheduleKanban/components/kanbanColumn.vue

```diff
@@ -89,16 +90,25 @@
           class="scrollbar tw-overflow-y-auto tw-overflow-x-hidden tw-mb-4 tw-h-full tw-px-2"
           handle=".dot-vertical"
           @end="changeDate"
-          :disabled="isBlank"
+          :disabled="isBlank && isWeekAgenda"
         >
           <kanban-card
+            v-if="!hideCard"
             v-for="(card, index) in cards"
             :id="card.id"
```

Comment:
Por favor si aca se envia el mismo props enviar is component y realizar un modelo  que cambie de componente